package.json

```diff
@@ -108,7 +109,7 @@
   },
   "lint-staged": {
     "*.ts": [
-      "eslint --cache --fix"
+      "yarn lint:fix"
```

Comment:
We should still use `--cache` so that we only lint changed files. https://eslint.org/docs/latest/use/command-line-interface#:~:text=%2D%2Dcache%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Only%20check%20changed%20files%20%2D%20default%3A%20false